app/src/main/res/values/strings.xml

```diff
@@ -116,4 +116,7 @@
     <string name="status_disconnecting">disconnecting</string>
     <string name="status_waiting">connecting in %d seconds</string>
 
+    <!-- Emoji message-->
+    <string name="msg_no_recent_emoji">No recent emoji</string>
```

Comment:
Add the following string for Brazilian translation:

`<string name="msg_no_recent_emoji">Nenhum emoji recente</string>`

Thanks.